board/safety/safety_volkswagen.h

```diff
@@ -0,0 +1,169 @@
+const int VW_MAX_STEER = 300;               // 3.0 nm
+const int VW_MAX_RT_DELTA = 130;            // max delta torque allowed for real time checks
+const uint32_t VW_RT_INTERVAL = 250000;     // 250ms between real time checks
+const int VW_MAX_RATE_UP = 10;
+const int VW_MAX_RATE_DOWN = 10;
+const int VW_DRIVER_TORQUE_ALLOWANCE = 100;
+const int VW_DRIVER_TORQUE_FACTOR = 4;
+
+int vw_ignition_started = 0;
+struct sample_t vw_torque_driver;           // last few driver torques measured
+int vw_rt_torque_last = 0;
+int vw_desired_torque_last = 0;
+uint32_t vw_ts_last = 0;
+
+// Safety-relevant CAN messages for the Volkswagen MQB platform.
+#define MSG_EPS_01              0x09F
+#define MSG_ACC_06              0x122
+#define MSG_HCA_01              0x126
+#define MSG_GRA_ACC_01          0x12B
+#define MSG_LDW_02              0x397
+#define MSG_KLEMMEN_STATUS_01   0x3C0
+
+static void volkswagen_init(int16_t param) {
+  UNUSED(param); // May use param in the future to indicate MQB vs PQ35/PQ46/NMS vs MLB, or wiring configuration.
+  controls_allowed = 0;
+  vw_ignition_started = 0;
+}
+
+static int volkswagen_ign_hook(void) {
```

Comment:
you can keep the comment (perhaps move it before the safety_hooks definitions), but I would delete this function until it's different from the default_hook, which you should use instead.

Reply:
Change accepted.